### DISCUSSION TITLE
How to send manufacturer  specific command directly within bluealsa/bluez?

### DISCUSSION THREAD

**Discussion by AlphaOx42** [Arkq/bluez-alsa#732] [Q&A]
Hi everyone,

I am currently using  a Infineon (Cypress)  module based on the CYW4343W chipset and I was experiencing audio dropouts when  establishing a new connection while streaming A2DP.

According to Infineon we must change the priority of the A2DP stream as [workaround](https://community.infineon.com/t5/Knowledge-Base-Articles/Bluetooth-Scan-Operation-and-A2DP-Streaming-in-CYW43xxx-Devices/ta-p/735062), which also seems to be working.
But currently I'm setting the priority using the following command: 
`hcitool -i hci0 cmd  0x3f 0x11a <connection handle> 0x01 0x01`

I'd prefer avoiding a running such command and directly set this priority from bluealsa/bluez when starting and stopping an A2DP stream. I'm not so confident in the bluealsa stack, so I am unsure where and how I can achieve this goal ?

Every help is welcome
cheers

**Comment by borine** [comment]
This appears to be related to an issue recently raised in Bluez: https://github.com/bluez/bluez/issues/722

Its not clear whether the Bluez team intend to progress any of the suggestions in there though.

**Reply by AlphaOx42** [reply]
Yes I saw this issue, I'd hoped there is another possibility.
Thank you for your feedback. I guess I'd to go the easier way and use hcitool.
